packages/perseus/src/widgets/definition.jsx

```diff
@@ -89,25 +49,38 @@ class Definition extends React.Component<DefinitionProps> {
     };
 
     render(): React.Node {
-        const content = (
-            <DefinitionContent
-                apiOptions={this.props.apiOptions}
-                content={this.props.definition}
-                key={this.props.definition}
-                trackInteraction={this.props.trackInteraction}
-                widgets={this.props.widgets}
-            />
-        );
-
         return (
-            // $FlowFixMe[incompatible-type]: content prop
-            <Tooltip content={content} placement="top">
-                <span className="perseus-widget-definition">
-                    <View style={styles.definitionLink}>
-                        {this.props.togglePrompt}
-                    </View>
-                </span>
-            </Tooltip>
+            <Popover
+                content={
+                    <PopoverContentCore
+                        color="white"
+                        style={styles.tooltipBody}
+                        closeButtonVisible={true}
+                    >
+                        <Renderer
+                            apiOptions={this.props.apiOptions}
+                            content={this.props.definition}
+                            widgets={this.props.widgets}
+                        />
+                    </PopoverContentCore>
+                }
+                placement="top"
+            >
+                {({open}) => (
+                    <span className="perseus-widget-definition">
```

Comment:
It'd be nice to eventually have all of our Perseus widgets have a consistent classname on their root HTML element (maybe we could even do this in the widget container that the Renderer wraps around all widgets). 

No action required!